tests/passes/unsharing/while.expect

```diff
@@ -7,13 +7,12 @@ component main(go: 1, clk: 1) -> (done: 1) {
     other = std_reg(32);
```

Comment:
Is this test sufficient to catch the regressions of a similar nature in the future or would it be helpful to commit the dahlia generated code from the failing example?

Reply:
It will catch this flavor of bug, if it arises again since the test has been changed to reflect the desired conservative assumption.